reposilite-site/data/guides/migration/artifactory.md

```diff
@@ -0,0 +1,38 @@
+---
+id: artifactory
+title: Artifactory
+---
+
+### Instructions
```

Comment:
I'd remove this header. Also, I think you can replace this ordered list with a standard block of text

Reply:
Working on this today. I removed the frontmatter, but left the header 3 as I think that may have been what you are intending. There are other headers in the document so I thought it made sense this way. If I'm backwards just let me know and I'll correct.

The technical writing nerd in me wanted to leave the ordered list because its generally good practice to provide separation of steps via an ordered list if possible for clarity, or an unordered list if order truly does not matter. Main goal being readability, following each step. Just wanted to share my thoughts on it first. Happy to change it if you would still prefer a raw paragraph of course. Your vote wins :+1:

Reply:
By the `header` I've meant the ` ### Instructions`, not the file metadata between `---` :P The metadata part is required, that's why the deployment failed.

I'd turn this from list into a standard block of text mostly because I think it would be just easier to read as there is a lot of text and it could be combined with the rest of notes I've made below. If you feel that your proposal is still a better choice for newcomers, I can merge this this way.